docs/PersonalApplications.rst

```diff
@@ -292,135 +236,134 @@ Experience
+|PRFL-15|
 
-From the drop-down menu of your display name, select My Activities. You
-will be redirected to your own activity stream page.
+To remove an experience, you can simply click on Remove Experience button.
 
-|image23|
+|PRFL-16|
 
-Here, you can see your activities (inside and outside a space) and
-activities where you were :ref:`mentioned <Mentioning-People>`, liked or left comments.
+    .. note:: After making changes, make sure you click **Save** to update your profile or **Cancel** to dismiss the changes.
 
-The activities in My Activities are similar to those in :ref:`My Activities <Managing-Activities>` of the Social Intranet homepage. 
 
-.. note:: Your activity stream page (``../activities/[username]``) is visible
-			to anyone even if they are not your connections. They can comment or
-			like on your activity stream page. However, only your connections
-			can see the Activity Composer that enables them to post on your stream.
+To add a new experience, you can simply click on Add new button in the top Work Experiences drawer, and a new empty experience form is added.
 
- .. _Managing-Connections:
+|PRFL-17|
 
-=========================
-Managing your connections
-=========================
+.. _Kudos:
 
+Weekly Kudos
+--------------
 
-Building connections between you and other users in the system is very
-important. It will not only help you follow their activities on the
-activity stream, but also improve your communication and collaboration.
+The number of Kudos received and sent by the user is displayed in the Kudos section.
 
-This section shows you how to make connection with your desired users
-via the following topics:
+|PRFL-5|
 
- * :ref:`Viewing profile of other contacts <View-Profile-Contacts>`
- * :ref:`Sending connection requests <Send-Connection-Request>`
- * :ref:`Revoking a connection request <Revoke-Connection-Request>`
- * :ref:`Accepting/Denying a connection request <Accept-Deny-Connection-Request>`
- * :ref:`Disconnecting from your contacts <Disconnect-Contact>`
+By clicking on Received or Sent Kudos number, a drawer is opened that displays the details of Kudos received or sent.
 
-To build your connections, you first need to click the hamburger menu on
-the side navigation bar and select **People**.
+|PRFL-6|
 
-|PEOPLE1|
+All Kudos in the drawer are clickable and open, in a new browser tab, the source activity where the Kudos was sent or received.
 
-The **People** page appears.
 
-    .. note:: If there are so many connections, the Show More bar will appear at the page bottom. Click Show More to see more connections.
-             If you don't have any connection, a simple text is displated *NO PEOPLE*.
+.. _Coins:
 
-.. __MyConnectionPage.Tabs:
+Coins
+--------
 
-The People page includes the following sections:
+Monthly Rewards block displays the number of coins earned through  points and through kudos (the same value we already have in the cumulated Rewards in My Wallet application).
```

Comment:
add link to My Wallet

Reply:
Done. New commit is added for adding the wallet link.